app/javascript/app/document-capture/components/documents-step.jsx

```diff
@@ -53,6 +53,6 @@ DocumentsStep.defaultProps = {
  *
  * @return {boolean} Whether step is valid.
  */
-DocumentsStep.isValid = (value) => Boolean(value.front_image && value.back_image);
+export const isValid = (value) => Boolean(value.front_image && value.back_image);
```

Comment:
It's technically valid to assign a property into an `instanceof Function`, but the [TypeScript JSDoc flavor](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html) doesn't handle this case very well to "re-open" the React component definition for extension. So I opted instead to make this a proper exported member of the module, which retains the benefit of colocality, while integrating a little better with type-checking.